components/facebook_groups/README.md

```diff
@@ -0,0 +1,7 @@
+# Getting Started
+You need to install the Pipedream app in your Facebook Group to use any prebuilt triggers or actions. To install Pipedream, navigate to the group where you're an administrator, and in the left sidebar, under **Settings** > **Group Settings**, you'll find **Manage Advanced Settings**. Click the Edit button under **Apps**.
```

Comment:
>to use any prebuilt triggers or actions

I thought it was, "to make API requests to Facebook Groups and use any pre-built trigger or actions"